studyhub/base/views.py

```
@@ -1,12 +1,44 @@
 
+def loginPage(request): # Don't call the login view Login(), that is a Django function and will cause conflict
+
+    if request.user.is_authenticated:
+        return redirect('home')
+
+    if request.method == 'POST':
+        username = request.POST.get('username')
+        password = request.POST.get('password')
+        try:
+            user = User.objects.get(username=username)
+        except:
```

Ruff

Do not use bare `except`

(E722)